node/declaration.ts

```diff
@@ -59,15 +60,20 @@ export const mergeDeclarations = (
 
   // Finally, we must create declarations for functions that are not declared
   // in the TOML at all.
-  for (const name in functionsConfig) {
-    const { cache, path } = functionsConfig[name]
+  for (const name in { ...internalFunctionsConfig, ...userFunctionsConfig }) {
+    const { cache, path } = internalFunctionsConfig[name] || userFunctionsConfig[name]
 
     // If we have a path specified, create a declaration for each path.
     if (!functionsVisited.has(name) && path) {
       const paths = Array.isArray(path) ? path : [path]
 
       paths.forEach((singlePath) => {
-        declarations.push({ cache, function: name, path: singlePath })
+        const declaration: Declaration = { function: name, path: singlePath }
+        // only add cache if not null or undefined
+        if (cache) {
```

Comment:
Feels like this comment could be encoded into the actual assertion?

```suggestion
        if (cache !== null && cache !== undefined) {
```